data_structures/binary_tree/fenwick_tree.py

```diff
@@ -64,13 +64,13 @@ def get_array(self) -> list[int]:
         """
         arr = self.tree[:]
         for i in range(self.size - 1, 0, -1):
-            j = self.next(i)
+            j = self.next_index(i)
             if j < self.size:
                 arr[j] -= arr[i]
         return arr
 
     @staticmethod
-    def next(index: int) -> int:
+    def next_index(index: int) -> int:
```

Comment:
I don't think this should be a problem as this can only be called by `self.next`, so it's not overriding the builtin `next` function. Is flake8 marking this?

Reply:
> I don't think this should be a problem as this can only be called by `self.next`, so it's not overriding the builtin `next` function. Is flake8 marking this?

Yeah, I just realised that it wouldn't be a problem. However, yes, `flake8` does pick up on this. So either we add a `noqa` or we rename it

Reply:
We should just ignore it on a project level. It doesn't make sense. I'm specifically talking about `A003`: https://github.com/gforcada/flake8-builtins/blob/master/flake8_builtins.py#L19. They're also planning to disable it by default (https://github.com/gforcada/flake8-builtins/pull/76).

A file called `.flake8` should be added at the root of the repository with the following content:
```ini
[flake8]
ignore =
    A003  # definition ...
```

Reply:
> We should just ignore it on a project level. It doesn't make sense. I'm specifically talking about `A003`: https://github.com/gforcada/flake8-builtins/blob/master/flake8_builtins.py#L19. They're also planning to disable it by default ([gforcada/flake8-builtins#76](https://github.com/gforcada/flake8-builtins/pull/76)).
> 
> A file called `.flake8` should be added at the root of the repository with the following content:
> 
> ```ini
> [flake8]
> ignore =
>     A003  # definition ...
> ```

Okay, I'll make this file, ignore `A003` and revert my function name changes